test/dsl/dsl_basics.jl

```diff
@@ -272,7 +282,7 @@ let
     @test rn == mixedsys
 end
 
-# Test variables that appear only in rates and aren't ps
+# Test that variables that appear only in rates and aren't ps
```

Comment:
Because I was going through the files from start to finish, I straightened out some of the writing while I was at it.